rel/server/env.sh.eex

```diff
@@ -4,6 +4,13 @@ if [ "$LIVEBOOK_CLUSTER" = "auto" ] && [ ! -z "$FLY_APP_NAME" ]; then
   if [ ! -z "${LIVEBOOK_NODE}" ];
     then export LIVEBOOK_NODE="${FLY_APP_NAME}-${FLY_IMAGE_REF##*-}@${FLY_PRIVATE_IP}";
   fi
+elif [ "$LIVEBOOK_CLUSTER" = "auto" ] && [ ! -z "$KUBERNETES_SERVICE_HOST" ]; then
+  export POD_NAME="${POD_NAME:-$(echo $HOSTNAME)}"
+  export POD_NAMESPACE="${POD_NAMESPACE:-$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace)}"
+  export LIVEBOOK_CLUSTER="dns:${POD_NAME}.${POD_NAMESPACE}.svc.cluster.local"
+  if [ ! -z "${LIVEBOOK_NODE}" ];
+    then export LIVEBOOK_NODE="${POD_NAME}@${POD_IP}";
```

Comment:
`POD_IP` isn't exported by default from `k8s` and I'm not sure if we should use `KUBERNETES_SERVICE_HOST`.

Reply:
[Allegedly](https://stackoverflow.com/a/34678916) the correct IP it should be retrievable from the network interface with `hostname -i`, so we can try that.

If it doesn't work, then we could check if POD_IP is present, and if not, we exit and print an error telling them to expose it in the config.